diet-error-handler/tmp/errors.log

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Are the error logs supposed to be part of this PR?

Reply:
Ha no I did not. Let me fix that.